### PR TITLE
nodejs/week2

### PR DESCRIPTION
**Pull request by arya56**

### CODE REVIEW

File: nodejs/week2/warmup/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    else{
+        res.send(`Please write a query in this format /numbers/multiply/11/12`)
+    }
+});
+app.get("/numbers/multiply/:first", (req, res) => {
+    res.send(`Please write parameters in this format /numbers/multiply/11/12`)
+});
+app.get("/numbers/multiply/", (req, res) => {
+    res.send(`parameters should add in this format /numbers/multiply/11/12`)
+});
```
Comment (khjcph) [inline nodejs/week2/warmup/index.js:47]: This is not strictly required by the homework.

File: nodejs/week2/warmup/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            res.send(`Multiply of ${number1} to ${number2} is ${multiply} `)
+        }
+        else{
+            res.send(`Please write numbers on the query`)
+        }
+    }
+    else{
+        res.send(`Please write a query in this format /numbers/multiply/11/12`)
+    }
+});
```
Comment (khjcph) [inline nodejs/week2/warmup/index.js:41]: Try to see if you can identify all the code that these two routes share / have in common. If you'd like to, try to extract that code into reusable functions.

File: nodejs/week2/meal-sharing/src/backend/api/meals-router.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
+const meals = require('./../data/meals.json');
+const express = require('express');
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+  try {
+    console.log('in meals');
```
Comment (khjcph) [inline nodejs/week2/meal-sharing/src/backend/api/meals-router.js:7]: remember to remove debugging :)

File: nodejs/week2/warmup/index.js (hunk 2 above)
Comment (arya56) [reply nodejs/week2/warmup/index.js]: Thanks @khjcph for this wise comment, I will work on it.

**Pull request merged**